docs/UserGuide.md

```diff
@@ -124,19 +120,34 @@ Expected outcome:
 
 ### Adding a Module : `add`
 
-Adds a new module.
+Description:
+
+ * Adds any Module of choice into the list of Current Modules in GradPad.
+
+ * User can add any module by specifying their desired Module Code (eg. CS1231),
+ Modular Credits (eg. 4), and Tags (eg. Core) along with the `add` command, in their
+ input.
+ 
+ * Module Code and Modular Credit fields are compulsory inputs but Tag fields are optional.
+ Multiple Tags are allowed per module.
```

Comment:
maybe can include that the add command prevents duplicate modules from being created? actually right.... as of now our application allows duplicates if they are not completely **equal**, should we address that?

Reply:
Good suggestion! Added in